File: Plugin/double_tap_plugin/src/main/java/com/kronos/doubletap/DoubleTabConfig.java
package com.kronos.doubletap;


import org.apache.http.util.TextUtils;

public class DoubleTabConfig {
    public String injectClassName = "";
    public String injectFunctionName = "";

    public static String ByteCodeInjectClassName = "";
    public static String ByteCodeInjectFunctionName = "";

    public void transform() {
        if (TextUtils.isEmpty(injectClassName)) {
            ByteCodeInjectClassName = "com/wallstreetcn/sample/DoubleTapCheck";
        } else {
            ByteCodeInjectClassName = injectClassName.replace(".", "/");
        }
        if (TextUtils.isEmpty(injectFunctionName)) {
            ByteCodeInjectFunctionName = "isNotDoubleTap";
        } else {
            ByteCodeInjectFunctionName = injectFunctionName;
        }
    }
}

File: Plugin/double_tap_plugin/src/main/java/com/kronos/doubletap/DoubleTabConfig.kt
package com.kronos.doubletap

open class DoubleTabConfig {
    var injectClassName = ""
    var injectFunctionName = ""
    fun transform() {
        if (injectClassName.isEmpty()) {
            ByteCodeInjectClassName = "com/wallstreetcn/sample/DoubleTapCheck"
        } else {
            ByteCodeInjectClassName = injectClassName.replace(".", "/")
        }
        if (injectFunctionName.isEmpty()) {
            ByteCodeInjectFunctionName = "isNotDoubleTap"
        } else {
            ByteCodeInjectFunctionName = injectFunctionName
        }
    }

    companion object {
        @JvmField
        var ByteCodeInjectClassName = ""

        @JvmField
        var ByteCodeInjectFunctionName = ""
    }
}